wiki/CHEATSHEET.md

```diff
@@ -259,7 +259,8 @@ What I want | How to get it
 --- | ---
 Collect to array | `MoreCollectors.toArray()`
 Collect to boolean array using the `Predicate` applied to each element | `MoreCollectors.toBooleanArray()`
-Collect to `EnumSet` | `MoreCollectors.toEnumSet()`
+Collect `Map.Entry` entries to `HashMap` | `MoreCollectors.entriesToMap()`
```

Comment:
I don't think we should expose that we are collecting to `HashMap`. Just write "Collect `Map.Entry` entries to map".